Makefile

```
@@ -397,7 +452,11 @@ depend:
 # DO NOT DELETE
 
 src/FMI_search.o: src/FMI_search.h src/bntseq.h src/read_index_ele.h
```

checkmake (0.3.2)

[warning] Target "src/FMI_search.o" should be declared PHONY.

(phonydeclared)